### DISCUSSION TITLE
Homepage SSH

### DISCUSSION THREAD

**Discussion by Lockie85** [tteck/Proxmox#3233] [Q&A]
Hey,

I'd like to SSH via Visual Code to change my yaml files. But I'm uncertain of the default password to do this, or perhaps with automatic login on for the LXC it isn't possible in which case how might I disable that and set a new password?

Many Thanks.

**Comment by SeiTaN** [comment]
Hi,

I did it with the info in this link https://github.com/tteck/Proxmox/discussions/385 to change password.

Regards.
